controller/scheduler-v2/scheduler.go

```diff
@@ -0,0 +1,292 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/gopkg.in/inconshreveable/log15.v2"
+	ct "github.com/flynn/flynn/controller/types"
+	"github.com/flynn/flynn/controller/utils"
+	"github.com/flynn/flynn/host/types"
+)
+
+type Scheduler struct {
+	utils.ControllerClient
+	utils.ClusterClient
+	log        log15.Logger
+	formations *Formations
+
+	jobs    *jobMap
```

Comment:
It seems both formations and the scheduler are holding the job state in two different formats (formations are holding type -> jobID -> job, and the scheduler just jobID -> job), is this necessary?